geonode/geoserver/signals.py

```diff
@@ -84,7 +84,7 @@ def geoserver_post_save(instance, sender, created, **kwargs):
     if isinstance(instance, Layer):
         instance_dict = model_to_dict(instance)
         payload = json_serializer_producer(instance_dict)
-        producer.geoserver_upload_layer(payload)
+        #producer.geoserver_upload_layer(payload)
```

Comment:
Woking on a serialisation method to convert polygon into json. Starting by evaluating the necessity of Kombu

Reply:
ok